### ISSUE TITLE
Invocation every 15s

### ISSUE THREAD

**Issue by Uatschitchun** [mediathekview/plugin.video.mediathekview#140]
Is it really needed to invoke the addon every 15s?

I do get these line in log constantly every 15s:
```
2019-12-16 10:26:49.744 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: loading setting definitions
2019-12-16 10:26:49.744 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: trying to load setting definitions from old format...
2019-12-16 10:26:49.746 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: loading setting values
2019-12-16 10:26:49.749 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: loading setting definitions
2019-12-16 10:26:49.749 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: trying to load setting definitions from old format...
2019-12-16 10:26:49.751 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: loading setting values
2019-12-16 10:26:49.752 T:1723794304   DEBUG: CSettingsManager: requested setting (downloadpath) was not found.
2019-12-16 10:26:49.754 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: loading setting definitions
2019-12-16 10:26:49.754 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: trying to load setting definitions from old format...
2019-12-16 10:26:49.756 T:1723794304   DEBUG: CAddonSettings[plugin.video.mediathekview]: loading setting values

```

**Comment by Uatschitchun** [comment]
Is it really needed to invoke mediathekview every 15s?

**Comment by codingPF** [comment]
Its the up to date check for the movie database.
Not sure if it makes sense to reduce this or make this a configuration setting.

**Comment by codingPF** [comment]
Released to official Kodi Repo in v0.6.4

**Comment by Uatschitchun** [comment]
Thx ;)
